.gitignore

```diff
@@ -9,7 +9,7 @@
 /nbproject
 /*.ipr
 /*.iws
-/*.iml
+**/*.iml
```

Comment:
-1 to merge this.

This has side effects: it ignores for example `/foo/src/main/resources/bar/example.iml` too.
That might be acceptable, but in any case this belongs in a a separate PR, in a separate discussion and if we do end up doing it, we need to do it **consistently** for all .gitignore files.

Reply:
Hi @ge0ffrey! Thx for looking at this PR. I added that because when I opened `optaplanner` on IntelliJ, I ended up with many "added" .iml files, which is not something we want.. I think. Other repositories have one .gitignore per maven module, always excluding `*.iml`. Idk what's the preferred approach here, but I'm assuming we don't want to commit `*.iml` files, right?

Anyway, I can revert those changes for this PR, of course.

Reply:
Hello @ge0ffrey would you please let us know your thoughts about TIago's comment?

Thanks!

Reply:
Ok, I'll revert the changes.